### PR TITLE
Fix clean up issue with sync tests

### PR DESCRIPTION
**Pull request by huminzhi**

### CODE REVIEW

**Comment by sgoldberg-sfdc** [comment]
Change to +setup looks good and will enable that suite to pass when executed as part of the Everything target.

File: reporter.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
```
Comment (kchitalia) [inline reporter.py]: We might not need this. I am trying to see if using grep will suffice. Hang tight :)

Comment (wmathurin) [inline reporter.py]: You could simply do
```shell
cat output | sed 's/.*write-file.*/skipped/'
```

**Comment by kchitalia** [comment]
👍

**Comment by bhariharan** [comment]
👍

**Pull request merged**